chromium_src/components/services/storage/dom_storage/local_storage_impl.h

```diff
@@ -38,16 +38,18 @@ class LocalStorageImpl : public mojom::LocalStorageControl {
   void ForceKeepSessionState() override;
 
  private:
-  const url::Origin* GetNonOpaqueOrigin(const url::Origin& origin, bool create);
+  const blink::StorageKey* GetStorageKeyWithNonOpaqueOrigin(
+      const blink::StorageKey& storage_key,
+      bool create);
 
   void ShutDownInMemoryStorage(base::OnceClosure callback);
 
   std::unique_ptr<LocalStorageImpl_ChromiumImpl> local_storage_;
   std::unique_ptr<LocalStorageImpl_ChromiumImpl> in_memory_local_storage_;
   mojo::Receiver<mojom::LocalStorageControl> control_receiver_{this};
   // LocalStorageImpl works only with non-opaque origins, that's why we use an
-  // opaque->non-opaque origin map.
-  std::map<url::Origin, url::Origin> non_opaque_origins_;
+  // map of opaque origins to StorageKeys with non-opaque origins.
```

Comment:
"an map" -> "a map"

Reply:
Fixed in https://github.com/brave/brave-core/pull/9558/commits/c9b9ca0b2d4f2a19cb045c440c06445841cee141